sqlite3/sqlite3.php

```diff
@@ -178,7 +178,7 @@ public function prepare($query) {}
      * @param string $query <p>
      * The SQL query to execute.
      * </p>
-     * @return SQLite3Result an <b>SQLite3Result</b> object if the query returns results. Otherwise,
+     * @return SQLite3Result|bool an <b>SQLite3Result</b> object if the query returns results. Otherwise,
```

Comment:
`bool` can be replaced with `false` here as according to  https://github.com/php/php-src/blob/master/ext/sqlite3/sqlite3.stub.php#L60 and https://github.com/php/php-src/blob/master/ext/sqlite3/sqlite3.c#L541 seems method can return `SQLite3Result|false` but can not return `true`